website/main.py

```diff
@@ -129,16 +155,39 @@ def update(attrname, old, new):
         target = data['sname'][selected_index]  # pulling out object name
 
         band, mags, magserr = genphot.main(target)  # grab photometric data fro given object
-        photcds.data['bands'] = band  # update photometric table with known bands
-        photcds.data['vals'] = mags  # update photometric table with corresponding magnitudes
-        photcds.data['valserr'] = magserr  # update photometric table with corresponding magnitude errors
+        photcds.data = dict(bands=band, vals=mags, valserr=magserr)
 
         inventory = geninfo.main(target)  # pull out full inventory (minus photometry) for given object
         infopre.text = inventory  # update the object info text box
 
         specfig.title.text = target  # change title on spectra to object name
         # TODO: add spectra in here
         return
+
+    def sbar_update(attrname, old, new):
+        """
+        Python callback function running in current document on search bar keypress
+        The parameters shown are not used in shown script but are required in the bokeh callback manager
+
+        Parameters
+        ----------
+        attrname
+            The changing attribute (required under hood)
+        old
+            The old object (required under hood)
+        new
+            The new object (required under hood)
+        """
+        qry = sbar.value_input
+        results = np.array(st.querying(qry))
+        snames, ras, decs = [], [], []
+        if results.size:  # not empty
+            snames = results[:, 0]
+            ras = results[:, 1]
+            decs = results[:, 2]
+        dtcds.data = dict(sname=snames, ra=ras, dec=decs)
```

Comment:
There may be ways to simplify this re-formatting of the data, depending on which format you request the results as.